ktor-server/ktor-server-plugins/ktor-server-double-receive/jvm/src/io/ktor/server/plugins/doublereceive/FileCacheJvm.kt

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2014-2022 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.server.plugins.doublereceive
+
+import io.ktor.util.cio.*
+import io.ktor.utils.io.*
+import kotlinx.coroutines.*
+import java.io.*
+import java.nio.*
+import kotlin.coroutines.*
+
+internal actual class FileCache actual constructor(
+    private val body: ByteReadChannel,
+    bufferSize: Int,
+    context: CoroutineContext
+) : DoubleReceiveCache {
+    private val file = File.createTempFile("ktor-double-receive-cache", ".tmp")
```

Comment:
Add `deleteOnExit()`?

Reply:
I would say it should be in `call.afterFinish`.

Reply:
Yes, that too